paper.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'CMinx: Restructured Text Documentation Generator for CMake'
```

Comment:
I would say the most important part of the title is that it's for CMake, so it would probably be best if we could reword it to move CMake closer to the beginning of the title. Maybe "CMinx: The CMake Documentation Generator"? Or "CMinx: Documenting CMake with Restructured Text"?

With the current title, readers might miss the CMake part and assume it's an alternative to already existing RST documentation generators